daphne_brain/settings.py

```diff
@@ -189,6 +189,15 @@
     'database': 'edldatabase'
 }
 
+ECLSS_DATABASE = {
+    'drivername': 'postgres',
+    'host': 'www.selva-research.com',
+    'port': '5432',
+    'username': 'eclss',
+    'password': 'eclsstest12345',
```

Comment:
I feel this shouldn't be in public code

Reply:
Not sure how to handle this otherwise.

Reply:
This can and should be handled with environment variables, check other important things in this file for that